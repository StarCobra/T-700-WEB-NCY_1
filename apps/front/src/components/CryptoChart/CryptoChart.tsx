import React from "react"

export default function CryptoChart() {
  return <p>theCryptoChart</p>
}
