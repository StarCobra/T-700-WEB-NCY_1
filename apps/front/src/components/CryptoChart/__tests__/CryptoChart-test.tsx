import React from "react";
import { act, create } from "react-test-renderer";
import CryptoChart from "../CryptoChart";

describe("Components > CryptoChart > CryptoChart", () => {
  const tests = [
    {
      description: "generic",
      props: {},
    },
  ];
  tests.forEach((test) => {
    it(`should render the component ${test.description}`, async () => {
      let root;
      await act(async () => {
        root = create(<CryptoChart {...test.props} />);
      });
      expect(root).toMatchSnapshot();
    });
  });
});
