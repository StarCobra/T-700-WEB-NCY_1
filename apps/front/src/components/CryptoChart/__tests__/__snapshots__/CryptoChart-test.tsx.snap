// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components > CryptoChart > CryptoChart should render the component generic 1`] = `
<p>
  theCryptoChart
</p>
`;
